src/directLine.ts

```diff
@@ -869,4 +885,40 @@ export class DirectLine implements IBotConnection {
             )
         )
     }
+
+    private commonHeaders() {
+
+
+        if (this._userAgent === '') {
```

Comment:
isn't this defined in the constructor?

Reply:
It's defined as a member property. Not explicitly set in the constructor. I think I'll update to initialize the value to null instead of an empty string.